src/__tests__/renderForm.test.js

```diff
@@ -0,0 +1,135 @@
+import {
+    queryByTestId,
+    fireEvent,
+    getByText,
+    getByTestId,
+    getAllByTestId,
+    queryByText,
+    waitForDomChange
+} from '@testing-library/dom';
+import '@testing-library/jest-dom/extend-expect';
+import { main } from '../scripts/main';
+
+const POSITION_TO_TEST_PREV_BTN = 4;
+
+function getExampleDOM() {
+    const div = document.createElement('div');
+    div.innerHTML = `
+    <div class="container" data-testid="container">
+    <article class="pagination-component" data-testid="pagination">
+        <nav ul="" class="pagination-component__container">
+            <ul class="pagination-component__pages"></ul>
+        </nav>
+    </article>
+    </div>
+    `;
+    main(div);
+    return div;
+}
+
+// afterEach(() => (document.querySelector('.container').innerHTML = ''));
+
+describe('Tests', () => {
+    let container;
+
+    beforeEach(() => {
+        const url = new URL(`?p=1`, window.location.href);
+        window.history.pushState('', '', url);
+        container = getExampleDOM();
+    });
+
+    test('Testing rendering', () => {
```

Comment:
poczytaj sobie jak powinny wyglądać opisy testów.
powinieneś to trochę rozszerzyć, np:
`should render correct number of pagination elements`
do następnego np:
`should render correct number of elements after click next button`
itd